app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="login_page">
```

Comment:
- [ ] This activity requires the modification of the views in devise for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly. Kindly ensure to improve upon the same

**Your login view:**
![image](https://user-images.githubusercontent.com/12745474/201062050-07171d34-c7df-4a71-bfcd-661185642cf3.png)

**Wireframe login view:**
![image](https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png)